democracy_club/apps/core/tests/test_markdown.py

```diff
@@ -0,0 +1,38 @@
+from markdown.test_tools import TestCase
```

Comment:
I'm not sure how useful this is - I could obviously take this much further. We are already testing valid html elsewhere.

Reply:
I added additional smoke tests but want I want to do is actually pass a markdown file as an arguement. However, I'm struggling to find examples.